services/repository-managers/src/test/java/org/sagebionetworks/manager/util/OAuthPermissionUtilsTest.java

```diff
@@ -0,0 +1,38 @@
+package org.sagebionetworks.manager.util;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.repo.model.ACCESS_TYPE;
+import org.sagebionetworks.repo.model.UnauthorizedException;
+import org.sagebionetworks.repo.model.oauth.OAuthScope;
+
+import com.google.common.collect.ImmutableList;
+
+class OAuthPermissionUtilsTest {
+
```

Comment:
Add an additional test that verifies that all the current ACCESS_TYPE enumeration values are covered so that if in the future someone adds a new ACCESS_TYPE value the test would fail and inform the developer to map it to a scope.

Reply:
We can put some exceptions if they are not used.

Reply:
Done.